lib/tokenizers/normalizer.ex

```diff
@@ -162,3 +170,10 @@ defimpl Inspect, for: Tokenizers.Normalizer do
     concat(["#Tokenizers.Normalizer<", to_doc(attrs, opts), ">"])
   end
 end
+
+defmodule Tokenizers.Normalizer.ByteLevel do
+  @doc """
+  Gets ByteLevel normalizer's alphabet.
+  """
+  defdelegate alphabet(), to: Tokenizers.Native, as: :normalizers_byte_level_alphabet
```

Comment:
I would move it to the Normalizer module as `byte_level_alphabet`, so we don't have this micromodule :)